### PR TITLE
Fixes azure sql expansion

### PR DESCRIPTION
**Pull request by anthonydresser**
The vscode merge broken the sql expansion in extension views. This fixes that.

I also changed how we handle getting data from extension which removes the need to make changes in vscode code and is simpler

### CODE REVIEW

File: src/sql/parts/objectExplorer/common/objectExplorerViewTreeShim.ts [hunk 1]
```diff
@@ -141,8 +135,8 @@ export class OEShimService implements IOEShimService {
 			label: {
 				label: node.label
 			},
-			icon: URI.parse(icon),
-			iconDark: URI.parse(iconDark),
+			// icon: URI.parse(icon),
```
Comment (kburtram) [inline src/sql/parts/objectExplorer/common/objectExplorerViewTreeShim.ts]: dead-code

File: src/sql/parts/objectExplorer/common/objectExplorerViewTreeShim.ts [hunk 2]
```diff
@@ -141,8 +135,8 @@ export class OEShimService implements IOEShimService {
...
+			// iconDark: URI.parse(iconDark),
```
Comment (MattIrv) [inline src/sql/parts/objectExplorer/common/objectExplorerViewTreeShim.ts]: commented code

File: src/sql/workbench/browser/parts/views/customView.ts [hunk 3]
```diff
@@ -362,6 +538,31 @@ interface ITreeExplorerTemplateData {
 	aligner: Aligner;
 }
 
+// todo@joh,sandy make this proper and contributable from extensions
```
Comment (MattIrv) [inline src/sql/workbench/browser/parts/views/customView.ts]: should this todo be deleted?

**Comment by anthonydresser** [comment]
I'll have to look into how vscode tests this areas since its UI only used by extensions.

**Comment by kburtram** [comment]
@anthonydresser if it's not straightforward to add tests in this area I'm fine going ahead with the fix as-is.  Though we should make note of that problem and resolve when possible.

**Pull request merged**